sdk/python/kfp/dsl/__init__.py

```diff
@@ -15,6 +15,6 @@
 
 from ._pipeline_param import PipelineParam
 from ._pipeline import Pipeline, pipeline, get_pipeline_conf
-from ._container_op import ContainerOp
+from ._container_op import ContainerOp, SideCar
```

Comment:
Given Argo uses 'Sidecar' may make sense to use small 'c' to show compatibility?